src/devtools/client/webconsole/components/App.js

```diff
@@ -211,41 +210,42 @@ class App extends Component {
     const filterBar = this.renderFilterBar();
     const consoleOutput = this.renderConsoleOutput();
     const jsterm = this.renderJsTerm();
-    const eager = this.renderEagerEvaluation();
+    // const eager = this.renderEagerEvaluation();
 
     return this.renderRootElement([
       filterBar,
       dom.div(
         { className: "flexible-output-input", key: "in-out-container" },
         consoleOutput,
         jsterm,
-        eager
+        // eager
       ),
       editorMode
         ? GridElementWidthResizer({
-            key: "editor-resizer",
-            enabled: editorMode,
-            position: "end",
-            className: "editor-resizer",
-            getControlledElementNode: () => window.jsterm.node,
-            onResizeEnd: width => dispatch(actions.setEditorWidth(width)),
-          })
+          key: "editor-resizer",
+          enabled: editorMode,
+          position: "end",
+          className: "editor-resizer",
+          getControlledElementNode: () => window.jsterm.node,
+          onResizeEnd: width => dispatch(actions.setEditorWidth(width)),
+        })
         : null,
     ]);
   }
 }
 
-const mapStateToProps = state => ({
-  notifications: getAllNotifications(state),
-  reverseSearchInputVisible: state.ui.reverseSearchInputVisible,
-  reverseSearchInitialValue: state.ui.reverseSearchInitialValue,
-  editorMode: state.ui.editor,
-  editorWidth: state.ui.editorWidth,
-  sidebarVisible: state.ui.sidebarVisible,
-  filterBarDisplayMode: state.ui.filterBarDisplayMode,
-  eagerEvaluationEnabled: state.prefs.eagerEvaluation,
-  autocomplete: state.prefs.autocomplete,
-});
+const mapStateToProps = state => {
+  return {
```

Comment:
Weird that we don't have selectors for these, but I suppose that's fine.

Reply:
agreed